kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -1136,9 +1136,11 @@ define([
                 const importFile = $(dataElem).attr('data-file-name');
                 if (stagingAreaViewer.bulkImportTypes.includes(importType)) {
                     if (!(importType in bulkMapping)) {
+                        const appInfo = stagingAreaViewer.uploaders.app_info[importType];
                         bulkMapping[importType] = {
-                            appId: stagingAreaViewer.uploaders.app_info[importType].app_id,
+                            appId: appInfo.app_id,
                             files: [],
+                            outputSuffix: appInfo.app_output_suffix,
```

Comment:
pass along the suggested output suffix to the bulk app cell, just like in the regular app cell.